core/domain/state_domain.py

```diff
@@ -159,30 +160,46 @@ def get_all_html_content_strings(self):
 
         outcome_html = self.outcome.feedback.html
         html_list = html_list + [outcome_html]
+        # NOTE TO DEVELOPERS:
+        # The rules_index_dict below is used to  igure out the assembly of the
+        # html in the rule specs.
+        # See issue: https://github.com/oppia/oppia/issues/9413. We cannot use
```

Comment:
Write this as "TODO(#9413): ..." and make it an imperative, explaining what exactly needs to be done. Put the "We cannot use..." section as part of the general comment above it.

Reply:
done